### PR TITLE
[Controller]: added health controller

### PR DESCRIPTION
**Pull request by TarasYurkovskyi**
### Added:

- Health Controller

### CODE REVIEW

File: src/main/java/com/example/accommodationbookingapp/controller/HealthController.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ResponseStatus(HttpStatus.OK)
+@RequestMapping("/health")
+public class HealthController {
+
```
Comment (vadymhrnk) [inline src/main/java/com/example/accommodationbookingapp/controller/HealthController.java:14]: redundant empty line

**Review by vadymhrnk — APPROVED** [review]
LGTM

**Review by annton-shakur — APPROVED** [review]
GJ!

**Review by MinoDentori — APPROVED** [review]
GJ!

**Pull request merged**